packages/desktop-client/src/components/settings/UI.tsx

```diff
@@ -19,12 +19,12 @@ export const Setting = ({ primaryAction, style, children }: SettingProps) => {
     <View
       className={`${css([
         {
-          backgroundColor: colors.n9,
+          backgroundColor: theme.pillBackground,
           alignSelf: 'flex-start',
           alignItems: 'flex-start',
           padding: 15,
           borderRadius: 4,
-          border: '1px solid ' + colors.n8,
+          border: '1px solid ' + theme.pillBorder,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏very different color here